scripts.js

```diff
@@ -0,0 +1,47 @@
+;(function() {
+    function sleep(ms) {
+        return new Promise(resolve => setTimeout(resolve, ms));
+    }
+
+    window.addEventListener('DOMContentLoaded', () => {
+
+        const feedbackOpenButton$ = document
+            .querySelector('.order__button')
+
+        const feedbackCloseButton$ = document
+            .querySelector('.feedback-form__close-button')
+
+        const feedbackForm$ = document
+            .querySelector('.feedback-form')
+
+        const feedbackFormInner$ = document
+            .querySelector('.feedback-form__inner')
+
+        const formsBackground$ = document
+            .querySelector('.forms__darkback')
+
+        feedbackOpenButton$
+            .addEventListener('click', () => {
+                feedbackFormInner$.style.animation = null;
+                feedbackForm$.style.display = 'block';
+                formsBackground$.style.display = 'block';
+                sleep(1).then(() => {
```

Comment:
If you set `0` as default for `ms` parameter for sleep there is you can use it like `sleep().then(...`.